### DISCUSSION TITLE
过来膜拜大神

### DISCUSSION THREAD

**Discussion by Mongoose110** [Ivysauro/CNRT#12] [General]
作为nfc行业人员，表示自己做的统计都没你做都好。

**Comment by Ivysauro** [comment]
> 作为nfc行业人员，表示自己做的统计都没你做都好。

个人也算是NFC爱好者了，网上也还有很多人在做这方面的统计，本项目也有借鉴，期待你的作品，大家共同学习和提高~

（话说今天本项目怎么突然这么活跃😂

**Reply by Mongoose110** [reply]
因为我同步给小伙伴们了哈哈哈哈

**Reply by Ivysauro** [reply]
谢谢宣传~

你们不会是移动支付网的大佬吧🤔

**Comment by peace-dove** [comment]
膜拜+1
